backend/infrahub/message_bus/messages/__init__.py

```diff
@@ -58,6 +60,8 @@
     "git.repository.import_objects": GitRepositoryImportObjects,
     "schema.migration.path": SchemaMigrationPath,
     "schema.validator.path": SchemaValidatorPath,
+    "refresh.git.clone": GitRepositoryClone,
+    "refresh.git.fetch": GitRepositoryFetch,
```

Comment:
Just a small comment on the naming here. We've always followed the naming conventions so that the class matches the routing key.

So following the current convention it would be:

```python
    "refresh.git.clone": RefreshGitClone,
    "refresh.git.fetch": RefreshGitFetch,
```

This might sound a bit off and it could be that we should find other names. In this case it would probably have been better if we hadn't already "used" `git.*.*` as a binding.

Reply:
I think it make sense to create a new binding for all git messages like `git.*.*` that would be reserved for the task-workers  
Currently all consumers even the API server are subscribing to `refresh.*.*`

Reply:
It's a bit different:

```python
class InfrahubMessageBus:
    DELIVER_TIMEOUT: int = 30 * 60  # 30 minutes
    worker_bindings: list[str] = [
        "check.*.*",
        "event.*.*",
        "finalize.*.*",
        "git.*.*",
        "refresh.webhook.*",
        "request.*.*",
        "send.*.*",
        "schema.*.*",
        "transform.*.*",
        "trigger.*.*",
    ]
    event_bindings: list[str] = ["refresh.registry.*"]
```

As it is not anything under `refresh.registry.*` would be sent to all worker types both API workers and Prefect workers. And anything under `git.*.*` would be sent to the competing work queue that gets pick up by the first Prefect worker available. Which means that in order to use anything under "git." for broadcasts we'd have to modify the existing routing (i.e. have a call to .unbind() for that queue at startup, alternatively if it's a fresh instance of RabbitMQ everything would be recreated from scratch)